main.cpp

```diff
@@ -46,13 +46,17 @@
 #include "TransactionHistory.h"
 #include "model/TransactionHistoryModel.h"
 #include "model/TransactionHistorySortFilterModel.h"
-#include "daemon/DaemonManager.h"
 #include "AddressBook.h"
 #include "model/AddressBookModel.h"
 
+// IOS exclusions
+#ifndef Q_OS_IOS
```

Comment:
Good idea. Didn't get it to work with the "DEFINES+=" method you used for the scanner for some reason. I'll try again some other day and create a new PR if i get it to work.